Example/ImageKit/AppDelegate.swift

```diff
@@ -15,7 +15,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
-        _ = ImageKit.init(publicKey: "your_public_key", imageKitEndpoint: "https://ik.imagekit.io/your_imagekit_id", transformationPosition: TransformationPosition.PATH, authenticationEndpoint: "http://localhost:8080/auth")
+        ImageKit.init(publicKey: "IK_PUBLIC_KEY", urlEndpoint: "https://ik.imagekit.io/IK_ID", transformationPosition: TransformationPosition.PATH)
```

Comment:
Please move the `IK_PUBLIC_KEY` and `https://ik.imagekit.io/IK_ID` to a constants file like done in android

Reply:
The constants file should look like this https://github.com/imagekit-samples/quickstart/pull/40/commits/75d79da378538eaa1ae8d9e1169b53e76c2514a5